docs/source/gbd2017_models/concept_models/vivarium_swissre_breastcancer/concept_model.rst

```diff
@@ -277,38 +282,93 @@ Scale-up of breast cancer screening coverage among insured population
++--------+--------+-------+----------+-----------+---------------------+-------------+-------------+
+| A      | Female | 30-69 | Yes      | either    | MRI, every year     | 91%         | 100%        |
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+                                             
+| B      |        | 30-44 | No       | Yes       | BUS, every year     | 73.7%       | 100%        |      
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+   
+| C      |        | 45-69 | No       | Yes       | MAM+BUS, every year | 93.9%       | 100%        |      
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+    
+| D      |        | 30-69 | No       | No        | MAM, every 2 years  | 84.8%       | 100%        |
++--------+        +-------+----------+-----------+---------------------+-------------+-------------+    
+| E      |        | <30   | either   | either    | No screening                                    |
+|        |        | or 70+|          |           |                                                 |
++--------+--------+-------+----------+-----------+-------------------------------------------------+
+| F      | Male   | any   | either   | either    | No screening                                    |
++--------+--------+-------+----------+-----------+-------------------------------------------------+
+| MAM: mammography; BUS: breast ultrasound                                                         |
++--------------------------------------------------------------------------------------------------+
+ 
+ 
+.. todo::
+  The breast cancer recommendation says: "Women aged 69 years and older and with an average risk of breast cancer should have the opportunity to continue screening as long as their overall health is good and they have a life expectancy of 10 years or longer." I'm wondering if this only applies to average risk women and that high risk 70+ women would continue to have MRI (follow branch A). Yongquan, do you see any recommendation that separates out average risk and high risk 70+ women?   
+
+
+.. note:: 
+  see :download:`breast cancer screening memo <breast_cancer_screening_memo.docx>` for more in depth explanation how modelling decisions were adpated from guidelines, as well as assumptions and limiations of these modelling decisions. 
+
 
-.. image:: breast_cancer_screening_tree_China.svg
+:underline:`II. Probability of attending screening`
 
+ - 1) All simulants will be due a screening according to their attributes in the decision tree
+ - 2) Probability of simulants attending their first due screening is 22.5% (95%CI 20.4-24.6%).
+ - 3) If a simulant attends a screening, they are 1.89 (95%CI 1.06-2.04) times more likely to attend their next screening than those who did not attend a  
+      screening. 
+
```

Comment:
It's an odds ratio but i haven't found the original 2x2 table because the full text in Chinese is inaccessible. I found another article by Wu et al, 2012 and posted the PDF in `#vivarium-csu-cancer` channel see if that can be an alternative reference.